calicoctl/commands/apply.go

```diff
@@ -92,40 +95,42 @@ Description:
 		return nil
 	}
 
-	results := common.ExecuteConfigCommand(parsedArgs, common.ActionApply)
-	log.Infof("results: %+v", results)
-
-	if results.FileInvalid {
-		return fmt.Errorf("Failed to execute command: %v", results.Err)
-	} else if results.NumHandled == 0 {
-		if results.NumResources == 0 {
-			return fmt.Errorf("No resources specified in file")
-		} else if results.NumResources == 1 {
-			return fmt.Errorf("Failed to apply '%s' resource: %v", results.SingleKind, results.ResErrs)
-		} else if results.SingleKind != "" {
-			return fmt.Errorf("Failed to apply any '%s' resources: %v", results.SingleKind, results.ResErrs)
-		} else {
-			return fmt.Errorf("Failed to apply any resources: %v", results.ResErrs)
-		}
-	} else if len(results.ResErrs) == 0 {
-		if results.SingleKind != "" {
-			fmt.Printf("Successfully applied %d '%s' resource(s)\n", results.NumHandled, results.SingleKind)
-		} else {
-			fmt.Printf("Successfully applied %d resource(s)\n", results.NumHandled)
-		}
-	} else {
-		if results.NumHandled-len(results.ResErrs) > 0 {
-			fmt.Printf("Partial success: ")
+	return file.Iter(parsedArgs, func(updatedArgs map[string]interface{}) error {
```

Comment:
This is the main change in each command.  I'm wrapping the command in this iter and using the "updatedArgs" in place of the parsedArgs.